### PR TITLE
fix: E2E dev documentation

### PR DESCRIPTION
**Pull request by adityabhatia**

### CODE REVIEW

File: docs/developer/creating-test-environment.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Set up the test environment
+# Set up test environment for local testing
```
Comment (jakobmoellersap) [inline docs/developer/creating-test-environment.md:1]:
```suggestion
# Set up test environment for local dual-cluster mode
```

File: docs/developer/creating-test-environment.md [hunk 2]
```diff
@@ -20,29 +20,25 @@ You can choose between either a single-cluster or a two-cluster setup.
...
-3. Create a Secret to access the cluster which acts as SKR:
+   Next, create a `Secret` to access the SKR cluster from KCP:
 
    Go to `https://github.com/kyma-project/lifecycle-manager` and run the following commands, to create a secret and apply it to the KCP Cluster:
 
    ```sh
    chmod 755 ./operator/config/samples/secret/k3d-secret-gen.sh
    ./operator/config/samples/secret/k3d-secret-gen.sh
    ```
-   > _**NOTE:**_ In the **single-cluster setup**, adjust your contexts for applying the secret using `KCP_CLUSTER_CTX` and `SKR_CLUSTER_CTX` - both should point to the same cluster.
```
Comment (jakobmoellersap) [inline docs/developer/creating-test-environment.md:41]: I think the note could stay but without saying they should point to the same cluster no?

Comment (adityabhatia) [reply docs/developer/creating-test-environment.md:41]: In single cluster mode, the secret is not required  :)

File: docs/developer/creating-test-environment.md (hunk 1 above)
Comment (adityabhatia) [reply docs/developer/creating-test-environment.md:1]: I think this documentation still addresses local testing for single cluster cases

**Pull request merged**